icu4c/source/test/testdata/rbbitst.txt

```diff
@@ -32,7 +32,7 @@
 #   [ICU4C]   source/test/testdata/rbbitst.txt
 #   [ICU4J]   main/tests/core/src/com/ibm/icu/dev/test/rbbi/rbbitst.txt
 #
-# ICU4C's copy is the master.  If any changes are made to ICU4J's copy, make sure they
+# ICU4C's copy is the primary.  If any changes are made to ICU4J's copy, make sure they
```

Comment:
please add the word "one" --> "ICU4C's copy is the primary one."

Reply:
Done